cluster/src/main/java/com/hartwig/pipeline/calling/structural/StructuralCaller.java

```diff
@@ -83,52 +81,37 @@ public String namespace() {
     public List<BashCommand> commands(final SomaticRunMetadata metadata, final Map<String, ResourceDownload> resources) {
         List<BashCommand> commands = new ArrayList<>();
         commands.add(new ExportVariableCommand("PATH", format("${PATH}:%s", dirname(new BwaCommand().asBash()))));
+        commands.add(new ExportVariableCommand("PATH",
+                format("${PATH}:%s", dirname(new VersionedToolCommand("samtools", "samtools", Versions.SAMTOOLS).asBash()))));
```

Comment:
Does the driver script set the path for `bwa` itself or is that no longer required somehow?